geonode_mapstore_client/client/themes/geonode/scss/_filter-by-extent.scss

```diff
@@ -10,7 +10,7 @@
     display: flex;
     flex-direction: column;
     width: 100%;
-    height: 100%;
+    height: 40vh;
```

Comment:
I don't think this class has an effect anymore because is commented in the component. imho I think we should update this class to use the correct style instead to comment it.